.github/gem-push.yml

```diff
@@ -0,0 +1,49 @@
+name: Ruby Gem
+
+on:
+  push:
+    branches: [main]
+    tags:
+      - v*
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby 2.7
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.7.x
+      - name: Install dependencies
+        run: bundle install
+      - name: Run tests
+        # Skipping rubocop for now
+        # run: bundle exec rake
+        run: bundle exec rspec spec
+  build:
+    name: Build + Publish
+    needs: test
+    if: startsWith(github.ref, 'refs/tags/v')
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby 2.7
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.7.x
+
+      - name: Publish to RubyGems
+        run: |
+          mkdir -p $HOME/.gem
+          touch $HOME/.gem/credentials
+          chmod 0600 $HOME/.gem/credentials
+          printf -- "---\n:rubygems_api_key: ${GEM_HOST_API_KEY}\n" > $HOME/.gem/credentials
+          gem build *.gemspec
+          gem push *.gem
+        env:
+          GEM_HOST_API_KEY: "${{secrets.RUBYGEMS_AUTH_TOKEN}}"
```

Comment:
Has this secret been added? I don't have access to it.

Reply:
@aydrian it has not been added. send it my way and i'll add.

Reply:
added:
![image](https://user-images.githubusercontent.com/266481/105534907-06e49380-5ca3-11eb-9c29-c1132d8ac571.png)